apps/fishing-map/features/vessel-groups/vessel-groups.slice.ts

```diff
@@ -80,7 +80,7 @@ export const fetchVesselGroupsThunk = createAsyncThunk<
     const vesselGroupsParams = {
       ...DEFAULT_PAGINATION_PARAMS,
       cache: false,
-      'logged-user': true,
+      // 'logged-user': true,
```

Comment:
we needed this to avoid admins or gfw-staff requesting all the public vessel-groups in the API, I think we need to get it back but only when needed